templates/community/_events.html.j2

```diff
@@ -1,11 +1,11 @@
 {% import 'utility.macros.j2' as utility %}
 
 {% macro show_talk(who) %}
-  {% set bird_list = config.birds|selectattr('author', 'equalto', who.author)|list or config.birds_former|selectattr('author', 'equalto', who.author)|list %}
-  {% set bird = bird_list[0] if bird_list else who.author %}
+  {% set pages = all_pages|filter_pages('bird', 'eq', who.author) %}
+  {% set page = pages|first if pages|length else none %}
 
   <li>
-    {{ utility.link_if(bird.name, ['/birds/#bird', who.author]|join('-')) }}
+    {{ utility.link_if(page.title or who.author, ['/birds/#bird', who.author]|join('-')) }}
```

Comment:
![screenshot 2017-02-01 14 20 06](https://cloud.githubusercontent.com/assets/552316/22522453/95fe0da4-e889-11e6-86bf-3b6f98b36061.jpg)

The link is also broken, since there's no more `/birds/#bird-claudina`.